Sources/SwiftSyntaxBuilder/HasTrailingComma.swift

```diff
@@ -10,7 +10,17 @@
 //
 //===----------------------------------------------------------------------===//
 
-public protocol HasTrailingComma {
```

Comment:
Needed to make it internal, otherwise all implementations should declare `trailingComma` as `public`, not sure if it's fine